infra/token-transfer-server/src/lib/transfer.js

```diff
@@ -229,12 +222,13 @@ async function executeTransfer(transfer, opts) {
     const event = {
       userId: user.id,
       action: eventAction,
-      data: JSON.stringify({
-        transferId: transfer.id
-      })
+      data: {
+        transferId: transfer.id,
+        failureReason: failureReason
```

Comment:
this is conditionally set at line 23 below so remove?

Reply:
👍